### ISSUE TITLE
source_id registration of MRI-ESM2.0

### ISSUE THREAD

**Issue by yukimotoster** [WCRP-CMIP/CMIP6_CVs#208]
aerosol = MASINGAR mk2r4
    atmosphere = MRI-AGCM3.5 (320 x 160 TL159; 80 levels; top level 0.01hPa)
    atmospheric_chemistry = MRI-CCM2.1
    institution_id = MRI
    label = MRI-ESM2.0
    land_ice = None
    land_surface = HAL 1.0
    ocean = MRI.COM4.2 (tripolar primarily 0.5 deg latitude/1 deg longitude with meridional refinement down to 0.3 deg within 10 degrees north and south of the equator; 61 levels; top grid cell 0-2 m)    
    ocean_biogeochemistry = None
    release_year = 2017
    sea_ice = MRI.COM4.2
    source_id = MRI-ESM2-0

**Comment by durack1** [comment]
@yukimotoster did you intend to close this issue? Or did you want the `MRI-ESM2-0` registered?

**Comment by yukimotoster** [comment]
Dear Paul,

I intend to register our model MRI-ESM2-0.

Best regards,

Seiji


> 2017/01/22 3:57、Paul J. Durack <notifications@github.com> のメール：
> 
> @yukimotoster did you intend to close this issue? Or did you want the MRI-ESM2-0 registered?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 



------------
Dr. Seiji Yukimoto
Climate Research Department
Meteorological Research Institute
1-1 Nagamine, Tsukuba, 305-0052 Japan
Phone: +81-29-853-8610, Fax: +81-29-855-2552

**Comment by durack1** [comment]
@yukimotoster those changes have been updated, please take a look at [CMIP6_source_id.html](http://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_source_id.html)

There are some fields that require further information from you before we close this issue, these include (values between # symbols require replacement):
```
ocean: MRI.COM4.2 (tripolar primarily 0.5 deg latitude/1 deg longitude; 61 levels; top grid cell #X-X# m)
land_surface: HAL (#VERSION INFO#)
```

**Comment by yukimotoster** [comment]
The required information are added, with some modification for the grid of the ocean model as follows.

ocean = MRI.COM4.2 (tripolar primarily 0.5 deg latitude/1 deg longitude with meridional refinement down to 0.3 deg within 10 degrees north and south of the equator; 61 levels; top grid cell 0-2 m) 

land_surface = HAL 1.0

**Comment by durack1** [comment]
@yukimotoster we have augmented the fields that we are capturing in our `source_id`registrations. Can you please add in the `activity_participation` and `nominal_resolution` fields missing in [your `MRI-ESM2-0` registration](https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_source_id.json#L2321-L2369) - there are some guidance notes for completing these fields found in [Model_registration_template.md](https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/.github/Model_registration_template.md)

**Comment by durack1** [comment]
Closed in #354
